django/cantusdb_project/main_app/tests/test_views.py

```diff
@@ -2920,82 +2919,18 @@ def test_post_error(self):
             "This field is required.",
         )
 
-    def test_suggest_one_folio(self):
-        fake_source = make_fake_source()
-        # create fake genre to match fake_chant_2
-        fake_R_genre = make_fake_genre(name="R")
-        fake_chant_3 = make_fake_chant(
-            source=fake_source,
-            cantus_id="333333",
-            folio="001",
-            c_sequence=3,
-        )
-        fake_chant_2 = make_fake_chant(
-            source=fake_source,
-            cantus_id="007450",  # this has to be an actual cantus ID, since
-            # next_chants pulls data from CantusIndex and we'll get an empty
-            # response if we use "222222" etc.
-            folio="001",
-            c_sequence=2,
-            next_chant=fake_chant_3,
-        )
-        fake_chant_1 = make_fake_chant(
-            source=fake_source,
-            cantus_id="111111",
-            folio="001",
-            c_sequence=1,
-            next_chant=fake_chant_2,
-        )
-
-        # create one more chant with a cantus_id that is supposed to have suggestions
-        # if it has the same cantus_id as the fake_chant_1,
-        # it should give a suggestion of fake_chant_2
-        fake_chant_4 = make_fake_chant(
-            source=fake_source,
-            cantus_id="111111",
-        )
-
-        # go to the same source and access the input form
-        url = reverse("chant-create", args=[fake_source.id])
-        response = self.client.get(url)
-        self.assertEqual(response.status_code, 200)
-        # only one chant, i.e. fake_chant_2, should be returned
-        self.assertEqual(1, len(response.context["suggested_chants"]))
-        self.assertEqual("007450", response.context["suggested_chants"][0]["cid"])
-        self.assertEqual(
-            fake_R_genre.id, response.context["suggested_chants"][0]["genre_id"]
-        )
-
-    def test_fake_source(self):
-        """cannot go to input form with a fake source"""
-        fake_source = faker.numerify(
+    def test_nonexistent_source(self):
```

Comment:
This addition is unrelated to the main PR right? Looks fine, just curious.

Reply:
yes. I came across it while addressing the rest of the tests, and was initially baffled by the method name. Once I figured out what it was meant to do, I figured it needed to be fixed.